.github/workflows/docker_compose_works.yml

```diff
@@ -16,13 +16,11 @@ jobs:
           -f docker-compose.yml \
           -f .docker/docker-compose.ci.yml \
           up -d
-      with:
-        secret_gh_pages_api_token: ${{ secrets.GH_PAGES_TOKEN }}
-        git_committer_email: dev@payex.com
-        git_committer_name: payex-dev
-        gh_pages_publishing_source: gh-pages
       env:
         GITHUB_ACTOR: payex-dev
         JEKYLL_GITHUB_TOKEN : ${{ secrets.GH_PAGES_TOKEN }}
         JEKYLL_ENV: production
-
+        secret_gh_pages_api_token: ${{ secrets.GH_PAGES_TOKEN }}
```

Comment:
Can we use the same `GITHUB_TOKEN` environment variable so we don't have to declare it twice?

```suggestion
```